### ISSUE TITLE
Methods to upload data to s3 and create manifest file for Sagemaker Ground Truth

### ISSUE THREAD

**Issue by kschelonka** [AllenInstitute/segmentation-labeling-app#2]
Manifest is a json file and its contents should conform to ~Jline format.~ JSON Lines format:
https://docs.aws.amazon.com/sagemaker/latest/dg/augmented-manifest.html
http://jsonlines.org/

<table class="wrapped confluenceTable"><colgroup><col/><col/><col/></colgroup><tbody><tr><th class="confluenceTh">Field</th><th class="confluenceTh">Type</th><th class="confluenceTh">Description</th></tr><tr><td colspan="1" class="confluenceTd">source-ref</td><td colspan="1" class="confluenceTd">str</td><td colspan="1" class="confluenceTd">S3 URI: location of the ROI mask png</td></tr><tr><td class="confluenceTd">video-source-ref</td><td class="confluenceTd">str</td><td class="confluenceTd">S3 URI: location of the 2p video for the ROI </td></tr><tr><td class="confluenceTd">max-source-ref</td><td class="confluenceTd">str</td><td class="confluenceTd">S3 URI: location of the maximum projection for the ROI</td></tr><tr><td class="confluenceTd">avg-source-ref</td><td class="confluenceTd">str</td><td class="confluenceTd">S3 URI: location of the average projection image for the ROI</td></tr><tr><td class="confluenceTd">trace-source-ref</td><td class="confluenceTd">str</td><td class="confluenceTd">S3 URI: location of the trace data for the ROI</td></tr><tr><td colspan="1" class="confluenceTd">roi-data-source-ref</td><td colspan="1" class="confluenceTd">str</td><td colspan="1" class="confluenceTd">S3 URI: location of ROI coordinate data</td></tr><tr><td colspan="1" class="confluenceTd">roi-id</td><td colspan="1" class="confluenceTd">int</td><td colspan="1" class="confluenceTd">Unique ID for the ROI from the training data table</td></tr><tr><td class="confluenceTd">experiment-id</td><td class="confluenceTd">int</td><td class="confluenceTd">Internal experiment ID associated with ROI</td></tr></tbody></table></div>
